pkg/handler/aws/uploadProfilePicture/main.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/CollActionteam/collaction_backend/auth"
+	"github.com/CollActionteam/collaction_backend/internal/uploads"
+	awsRepository "github.com/CollActionteam/collaction_backend/pkg/repository/aws"
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/aws/aws-sdk-go/aws/session"
+)
+
+func handler(ctx context.Context, req events.APIGatewayV2HTTPRequest) (res events.APIGatewayV2HTTPResponse, err error) {
+	usrInf, err := auth.ExtractUserInfo(req)
+	if err != nil {
+		res = events.APIGatewayV2HTTPResponse{StatusCode: http.StatusForbidden, Body: "user not authorized"}
+		return res, err
```

Comment:
use the utils function here. check utils/api.go

Reply:
I followed the newer convention of the refactoring process so I am not sure if we would still need that because the previously merged PR @mrsoftware  serving as template had this format. So if we can discuss as a team we can have more clarity on these. @rubenhorn

Reply:
see response on other comment